test/federated/test_federated_client.py

```diff
@@ -61,14 +52,17 @@ def test_set_obj_other():
     assert fed_client._objects[dummy_data.id] == dummy_data
 
 
-def test_fit():
+@pytest.mark.skip(reason="bug in pytorch version 1.1.0, jit.trace returns raw C function")
+def test_fit(hook):  # pragma: no cover
+    fed_client = sy.VirtualWorker(hook=hook, id="test_fit_fc")
+
     data = torch.tensor([[-1, 2.0], [0, 1.1], [-1, 2.1], [0, 1.2]], requires_grad=True)
-    target = torch.tensor([[1], [0], [1], [0]])
+    target = torch.tensor([[1], [0], [1.0], [0]], requires_grad=True)
```

Comment:
why do you want the targets to be float?

Reply:
when it was an int I got an error.

Reply:
What error do you get? requires_grad=True requires a float tensor, but we don't need the gradient with respect to the target.